make/golang-v1.mk

```diff
@@ -34,55 +37,15 @@ endef
 # FGT is a utility that exits with 1 whenever any stderr/stdout output is recieved.
 FGT := $(GOPATH)/bin/fgt
 $(FGT):
-	go get github.com/GeertJohan/fgt
+	go get github.com/GeertJohan/fgt@262f7b11eec07dc7b147c44641236f3212fee89d
```

Comment:
For `fgt`, we are unlikely to ever update this tool, since it is a simple and complete tool already. So it is defense against the tool changing in the future.